deploy-disconnected-registry/common.sh

```diff
@@ -95,7 +95,7 @@ export REGISTRY_CONFIG=config.yml
 
 export SOURCE_PACKAGES='quay-operator,kubernetes-nmstate-operator,metallb-operator,ocs-operator,local-storage-operator,advanced-cluster-management'
 export PACKAGES_FORMATED=$(echo ${SOURCE_PACKAGES} | tr "," " ")
-export EXTRA_IMAGES=('quay.io/jparrill/registry:3' 'registry.access.redhat.com/rhscl/httpd-24-rhel7:latest')
+export EXTRA_IMAGES=('quay.io/jparrill/registry:3' 'registry.access.redhat.com/rhscl/httpd-24-rhel7:latest' 'quay.io/ztpfw/ui:latest')
```

Comment:
This will probably require to create a new ORG in Quay alongside ocatopic, olm, etc

Reply:
I need to rebuild the image, so we can postpone it